python/package/streamsx/messagehub/_messagehub.py

```diff
@@ -75,11 +60,11 @@ def publish(stream, topic, credentials=None, name=None):
     else:
         raise TypeError(schema)
 
-    _op = MessageHubProducer(stream, appConfigName=credentials, topic=topic)
+    _op = _MessageHubProducer(stream, appConfigName=credentials, topic=topic)
     _op.params['messageAttribute'] = _op.attribute(stream, msg_attr)
     
-class MessageHubConsumer(streamsx.spl.op.Source):
-    def __init__(self, topology, schema, vmArg=None, appConfigName=None, clientId=None, messageHubCredentialsFile=None, outputKeyAttributeName=None, outputMessageAttributeName=None, outputTimestampAttributeName=None, outputOffsetAttributeName=None, outputPartitionAttributeName=None, outputTopicAttributeName=None, partition=None, propertiesFile=None, startPosition=None, startTime=None, topic=None, triggerCount=None, userLib=None, name=None):
+class _MessageHubConsumer(streamsx.spl.op.Source):
+    def __init__(self, topology, schema, vmArg=None, appConfigName=None, clientId=None, messageHubCredentialsFile=None, outputKeyAttributeName=None, outputMessageAttributeName=None, outputTimestampAttributeName=None, outputOffsetAttributeName=None, outputPartitionAttributeName=None, outputTopicAttributeName=None, partition=None, propertiesFile=None, startPosition=None, startTime=None, topic=None, triggerCount=None, userLib=None, consistentRegionAssignmentMode=None, groupId=None, name=None):
```

Comment:
def __init__(self, topology, schema, vmArg=None, appConfigName=None, clientId=None, messageHubCredentialsFile=None, outputKeyAttributeName=None, outputMessageAttributeName=None, outputTimestampAttributeName=None, outputOffsetAttributeName=None, outputPartitionAttributeName=None, outputTopicAttributeName=None, partition=None, propertiesFile=None, startPosition=None, startTime=None, topic=None, triggerCount=None, userLib=None, groupId=None, name=None):

Please remove `consistentRegionAssignmentMode`. There is no such operator paramerter (It used to be one in an early alpha release).

Reply:
Darn, I removed the adding to params but not in `__init__`, good catch!

Reply:
@RolefH Parameter removed.